src/localization/making_string_changes.md

```diff
@@ -89,17 +89,19 @@ In this scenario, the only way to ensure that obsolete translations are ignored
 
 Things become a lot more complicated for Firefox:
 * Each localization is stored in a separate Mercurial repository.
-* The `en-US` locale is stored in the code repository (e.g. `mozilla-central`). That locale is used by the build system to build the US English version of Firefox, but it’s not used as source by the localization toolchain. In fact, in order to ship all versions of Firefox (Nightly, Beta, Developer Edition, Release, ESR) from a single l10n repository, we rely on a special [cross-channel repository](https://firefox-source-docs.mozilla.org/l10n/crosschannel/index.html) called `gecko-strings`, which includes all strings for all shipping versions of Firefox.
+* The `en-US` locale is stored in the code repository (e.g. `mozilla-central`). That locale is used by the build system to build the US English version of Firefox, but it’s not used as source by the localization toolchain. In fact, in order to ship all versions of Firefox (Nightly, Beta, Developer Edition, Release, ESR) from a single l10n repository, we rely on a special unified [repository](https://https://github.com/mozilla-l10n/firefox-l10n-source) in GitHub called `firefox-l10n-source`, which includes all strings for all shipping versions of Firefox.
 
 This is the scenario before the string change. Once again, although it already looks very busy, it’s a simplified high level view of the actual system.
 
+**Note**: The l10n repository transitioned to `firefox-l10n-source`, but was previously hosted in a Mercurial repository called `gecko-strings`. Images below use the former repository name.
```

Comment:
Let's fix the images instead. I will upload a new version to your branch.

Reply:
Thanks, you're right that works better.